weed/server/filer_server_handlers_write_autochunk.go

```diff
@@ -121,6 +126,10 @@ func (fs *FilerServer) doPutAutoChunk(ctx context.Context, w http.ResponseWriter
 		contentType = ""
 	}
 
+	if err := fs.checkPermissions(ctx, r, ""); err != nil {
```

Comment:
Is passing this empty string intentional?

Reply:
I think it is a mistake, pr: https://github.com/seaweedfs/seaweedfs/pull/6142